tools/webpack

```diff
@@ -46,6 +49,17 @@ def build_for_dev_server(host, port, minify, disable_host_check):
     if disable_host_check:
         webpack_args.append('--disable-host-check')
 
+    cwd = os.getcwd()
+    inotify_broken = False
+    with open('/proc/self/mounts') as mounts:
+        for line in mounts:
+            fields = line.split()
+            if fields[1] == cwd:
+                inotify_broken = fields[2] in ["nfs", "vboxsf"]
+    if inotify_broken:
+        webpack_args.append('--watch-poll=1000')
```

Comment:
I suspect 1000 is likely to be annoyingly high.  We might want to do like 500.

Reply:
Perhaps you don’t understand just how much CPU usage we’re talking about. It’s bad enough that my VBoxHeadless idles at ~5% CPU when running just background services (RabbitMQ and Redis are mostly responsible for that), but it jumps to ~20% with an idle `tools/webpack --watch` polling at 1000ms, or ~30% for an idle `tools/webpack --watch` polling at 500ms. That’s _before_ you add on the aggressively polling Tornado process for an additional ~20%.

Meanwhile, a hot-reloaded change already takes multiple seconds to become visible, so the 250ms expected difference in responsiveness really isn’t noticeable.

If you’re annoyed, use Docker, where inotify isn’t broken.

Reply:
Yeah, I guess I hadn't thought about the fact that because Virtualbox-on-macOS is already really slow, webpack's actual restart time is a problem on that platform anyway.

As a sidenote, I feel like we still need to figure out why `webpack` compilation takes several seconds of full CPU usage even when we haven't made any changes (e.g. when restarting `run-dev.py` manually); it seems excessive in a situation where in theory everything should have been cached.